src/net/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod if_;
```

Comment:
could you add a tiny comment explaining the name is to avoid the keyword issue? (I'm assuming that's why, anyway)